app/controllers/concerns/verify_setup_completed.rb

```diff
@@ -0,0 +1,36 @@
+module VerifySetupCompleted
+  extend ActiveSupport::Concern
+
+  MANDATORY_CONFIGS = %i[
```

Comment:
Removed defaults that are not added here:

* `shop_config`: checked, it seems this is generally gated via `if SiteConfig.shop_url.present?`